cli/src/org/partiql/cli/functions/WriteFile.kt

```diff
@@ -33,7 +33,7 @@ import java.io.OutputStreamWriter
 internal class WriteFile(valueFactory: ExprValueFactory) : BaseFunction(valueFactory) {
     override val signature = FunctionSignature(
         name = "write_file",
-        requiredParameters = listOf(StaticType.STRING, StaticType.STRING),
+        requiredParameters = listOf(StaticType.STRING, StaticType.ANY),
```

Comment:
`StaticType.STRING` is redundant when `StaticType.ANY` is present, since `StaticType.ANY` by definition includes the string type.

Consider making this just:
```
requiredParameters = listOf(StaticType.ANY),
```

Reply:
I think we need both of `StaticType.STRING` and `StaticType.ANY`, because they refer to how many arguments this function has.

Reply:
Unsure what you mean -- but lmk if below is correct:

`listOf()` returns the 2 required parameters (1: file path, 2: content to write). In our case, the file path needs to be a string, and the second argument (content), can be anything that our query returns (bags, lists, etc).

Reply:
Oh.  Wow I misread that.  Never mind.  Resolving.